src/state_machine/coordinator/mod.rs

```diff
@@ -42,6 +43,9 @@ pub enum Error {
     /// A bad sign_iter_id in received message
     #[error("Bad sign_iter_id: got {0} expected {1}")]
     BadSignIterId(u64, u64),
+    /// A malicious signer sent the received message
```

Comment:
Might be worthwhile documenting within the code itself what constitutes a malicious signer so when people make the rust docs, its easier to understand.

Reply:
We document it in the error messages:

```
                            warn!("Timeout gathering signature shares for signing round {} iteration {}", self.current_sign_id, self.current_sign_iter_id);
                            for signer_id in &self.sign_wait_signer_ids {
                                warn!("Mark signer {} as malicious", signer_id);
                                self.malicious_signer_ids.insert(*signer_id);
                            }
```

Reply:
Where do you think it should be documented @jferrant ?

Reply:
A signer is also considered malicious if they sent a bad `SignatureShare`, but that isn't implemented yet.